frontend/test/metabase/scenarios/embedding/embedding-linked-filters.js

```diff
@@ -0,0 +1,77 @@
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { PEOPLE } = SAMPLE_DATABASE;
+
+export const questionDetails = {
+  name: "Count of People by State (SQL)",
+  native: {
+    query:
+      'SELECT "PUBLIC"."PEOPLE"."STATE" AS "STATE", count(*) AS "count" FROM "PUBLIC"."PEOPLE" WHERE 1=1 [[ AND {{city}}]] [[ AND {{state}}]] GROUP BY "PUBLIC"."PEOPLE"."STATE" ORDER BY "count" DESC, "PUBLIC"."PEOPLE"."STATE" ASC',
+    "template-tags": {
+      city: {
+        id: "6b8b10ef-0104-1047-1e1b-2492d5954555",
+        name: "city",
+        "display-name": "City",
+        type: "dimension",
+        dimension: ["field", PEOPLE.CITY, null],
+        "widget-type": "category",
```

Comment:
If I remember correctly, this gets auto-corrected in the backend, but let's use the modern style
```suggestion
        "widget-type": "string/=",
```

Reply:
Great finding! Thank you.
Forgot to update it from the old test.